submissions/kuslosolodka/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,80 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  species: "dog",
+  name: "Brovko",
+  gender: "male",
+  legs: 4,
+  hands: undefined,
+  saying: "woof-woof!",
+};
+const cat = {
+  species: "cat",
+  name: "Murka",
+  gender: "female",
+  legs: 4,
+  hands: undefined,
+  saying: "meow!",
+};
+const woman = {
+  species: "human",
+  name: "Stefania",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: "I am Stefania!",
+};
+const man = {
+  species: "human",
+  name: "Marko",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  saying: "I am Marko!",
+};
+const cat_woman = {
+  species: "supernatural",
+  name: "Batgirl",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: cat["saying"],
+};
+dog.friend = `${woman.name},${man.name},${cat.name}`;
+cat.friend = undefined;
```

Comment:
`undefined` is a default value for some variable that was not given a value or for places where we wait for a value, but don't get it as a result of some mistake or something like that. If you want to specify something as "nothing" or "empty" use `null`. But of course, the end user of the app/website should see neither `null` nor `undefined`. You release your `undefined` to the eyes of the user, and that is a mistake. For the sake of this task, animals are allowed to have `0` hands, for example. And if someone doesn't have `friends` - you should just ignore that property.